chapters/6-analysis.tex

```diff
@@ -171,10 +177,9 @@ \subsection{Integrating different data sources}
 and make sure you understand their options and outputs.\index{merging data}
 
 % How to think about merges
-Chapter 3 discussed how to merge datasets
-based on their units of observation and identifying variables.
 When writing the code to implement merge operations,
-a few steps can help avoid mistakes.
+a few steps can help avoid mistakes.\sidenote{\url{
+		https://dimewiki.worldbank.org/Data\_Integration}}
 First, before writing code to combine the datasets,
 write pseudocode to understand which observations you expect to be
 matched or not, and why.
```

Comment:
this should also refer back to the data linkage table, as merge outcome expectations should be clear from information there, e.g. which source will have leftover observations, whether merge will be 1-1 or m-1 or 1-m

Reply:
done in 7e4080b820ac09645634ab0a33125a5ed9bdec9c